### ISSUE TITLE
StereoPannerNode

### ISSUE THREAD

**Issue by benji6** [mohayonao/web-audio-test-api#20]
StereoPannerNode is in the latest Web Audio API Editor's Draft (http://webaudio.github.io/web-audio-api) and is implemented in Firefox and Chrome, suggest we add it to web-audio-test-api?

**Comment by benji6** [comment]
#21

**Comment by mohayonao** [comment]
StereoPannerNode has not been supported in Safari8/iOS8 yet.

I want to switch enable/disable of new API features by configuration.

like below

```js
WebAudioTestAPI.config({
  targets: { chrome: 43, firefox: 38 },
  features: { promiseBasedDecodeAudioData: true },
});
```

But, I have not consider how to do it still.

**Comment by benji6** [comment]
The data here may be useful: https://browsercompat.herokuapp.com

In the meantime though I think there is a need to test newer features of the API (I definitely need to), so maybe we could add the StereoPannerNode and leave compatibility concerns to the user until a configuration method is implemented?

**Comment by mohayonao** [comment]
I've partially implemented new API support on v0.3.1.

You can use StereoPannerNode by configuration below.

```js
WebAudioTestAPI.setState("AudioContext#createStereoPanner", "enabled");
```

**Comment by benji6** [comment]
Thanks that's really helpful :+1:
